lib/generators/sufia/templates/migrations/change_featured_work_generic_file_id_to_generic_work_id.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeFeaturedWorkGenericFileIdToGenericWorkId < ActiveRecord::Migration
+  def change
+    return unless column_exists?(:featured_works, :generic_file_id)
+    rename_column :featured_works, :generic_file_id, :generic_work_id
```

Comment:
Ah! I see. Maybe part of this commit needs to be reverted: https://github.com/projecthydra/sufia/commit/2cb6ef129d1712fde5c2f6fc4d028b203f230805#diff-5bb87ab96f0bcf36eb184441813db0e7L5